lightning/src/ln/channelmanager.rs

```diff
@@ -3131,16 +3246,16 @@ where
 		};
 
 		peer_state.pending_msg_events.push(events::MessageSendEvent::SendFundingCreated {
-			node_id: chan.get_counterparty_node_id(),
+			node_id: chan.context.get_counterparty_node_id(),
 			msg,
 		});
-		match peer_state.channel_by_id.entry(chan.channel_id()) {
+		match peer_state.outbound_v1_channel_by_id.entry(chan.context.channel_id()) {
```

Comment:
Looking forward, that further split makes sense and we might introduce enums for the maps if it makes that easier but not super necessary.

I think I will revert to having `OutboundV1Channel::funding_signed` on `Channel::funding_signed` so that at least we have temp_chan_id and chan_id consistency in the maps. Adding an extra state in between will clear be a better solution. Luckily there's room to do that in a further PR.